cmake/Platform/Windows/Packaging/Tests/test_installer.py

```diff
@@ -97,7 +97,7 @@ def test_compile_project_fixture(test_create_project_fixture, context):
     launcher_target = f"{project_name}.GameLauncher"
 
     # configure
-    result = context.run([str(cmake_path),'-B', str(context.project_build_path), '-S', '.', '-G', 'Visual Studio 16 2019'], cwd=context.project_path)
+    result = context.run([str(cmake_path),'-B', str(context.project_build_path), '-S', '.', '-G', 'Visual Studio 17 2022'], cwd=context.project_path)
```

Comment:
You could remove the Visual Studio generator and would use the latest installed on Jenkins.

Reply:
I originally didn't go that route because I didn't want it to be unexpected when someone installs a newer version of VS on Jenkins but maybe that's lower maintenance.

Reply:
Well as Jenkins is a controlled environment, I imagine the person who installs a newer version of Visual Studio whenever it comes out will notice when if the configure fails due to the MSVC version.